airflow/jobs/backfill_job.py

```diff
@@ -203,18 +204,18 @@ def _update_counters(self, ti_status, session=None):
                 self.log.debug("Task instance %s succeeded. Don't rerun.", ti)
                 ti_status.running.pop(key)
                 continue
-            elif ti.state == State.SKIPPED:
+            if ti.state == State.SKIPPED:
```

Comment:
why did we change it from elif?

Reply:
pylint suggests it and it's a good suggestion because we have continues in those ifs which makes elif unnecessary